projects/vercel.com/pkg/package.yml

```diff
@@ -0,0 +1,41 @@
+distributable:
+  url: https://github.com/vercel/pkg/archive/refs/tags/{{version.raw}}.tar.gz
+  strip-components: 1
+
+versions:
+  github: vercel/pkg/tags
+
+dependencies:
+  #FIXME I don't know which versions it supports, but Node.js v18 is one of the most well-supported ones.
+  nodejs.org: ^18
```

Comment:
might be safer to use '*' so it doesn't cause conflicts (unless it's not ready for 21, in which case <21 might be better.)